### ISSUE TITLE
Is it possible to resize image?

### ISSUE THREAD

**Issue by DKWLB** [editor-js/image#25]
I get an image - all is ok, but want to make it inside a paragraph. To bypass an image with some text. Is it possible?

**Comment by BomdiZane** [comment]
I'd like to see this too. Usually when rendering the image I set the width and height to match my containers. Would be good user experience if they can see the same width and height during edit mode so they have a clear picture of what the final results would be.

**Comment by sebolio** [comment]
no answer after a year? why is open source always like this?

**Comment by findela** [comment]
the developers goes to sleeping mood or unable to find a answer till day.

**Comment by AlejandroMilan** [comment]
still no answer?

**Comment by vizvasrj** [comment]
o m g no answers
what the ffffffff

**Comment by neSpecc** [comment]
No, it is not possible. You can only make it stretched to the full width of the container.

**Comment by Sisableng** [comment]
still no solutions?

**Comment by xKZL** [comment]
;(

**Comment by bettysteger** [comment]
https://github.com/codex-team/editor.js/discussions/1938

**Comment by atstockland** [comment]
It would be sweet to just add a "dimensions" data attribute (alongside the caption field).  In that field one could supply dimensions like "100x100" and then pull ['data']['file']['dimensions'] to set in the html img tag.

**Comment by Sisableng** [comment]
> No, it is not possible. You can only make it stretched to the full width of the container.

after all these years I think this is good logic.

dev, please close this issue.

**Comment by Legendary4226** [comment]
Nothing is impossible :)
Juste adding the setting to chose the percentage of the container or a fixed value isn't much no ?
Weird

**Comment by GuilhermeAGouveia** [comment]
try this package: https://www.npmjs.com/package/editorjs-image-resize-crop

**Comment by pasichDev** [comment]
use mobile plugin
https://github.com/pasichDev/editorjs-image-tune-plus
